src/language/languageUnderstanding.js

```diff
@@ -88,6 +89,40 @@ class languageUnderstanding extends commonService {
 
     };
 
+    /**
+     * Trains app for that version. All changes since last training are applied.
+     * @returns {Promise.<object>}
+     */
+    train({parameters, headers, body}) {
```

Comment:
Please add the possible values for `parameters` listed here: https://westus.dev.cognitive.microsoft.com/docs/services/5890b47c39e2bb17b84a55ff/operations/5890b47c39e2bb052c5b9c45. And also get rid of `headers`.